.github/workflows/soundness.yml

```diff
@@ -51,6 +51,14 @@ on:
         type: string
         description: "Container image for the shell check job. Defaults to latest Swift Ubuntu image."
         default: "swift:6.0-noble"
+      yamllint_enabled:
+        type: boolean
+        description: "Boolean to enable the YAML lint job. Defaults to true."
+        default: true
+      yamllint_container_image:
```

Comment:
@shahmishal Would you like to have a separate option for the base image of `yamllint`, `spellcheck` and `flake8`? Or should all of this be one single options?